pkg/utils/schema.go

```diff
@@ -104,3 +104,10 @@ func IsSysDB(dbLowerName string) bool {
 func TemporaryDBName(db string) model.CIStr {
 	return model.NewCIStr("__TiDB_BR_Temporary_" + db)
 }
+
+// GetSysDBName get the original name of system DB
+func GetSysDBName(tempDB model.CIStr) (string, bool) {
+	ok := strings.Contains(tempDB.O, "__TiDB_BR_Temporary_")
+	i := strings.LastIndex(tempDB.O, "_")
+	return tempDB.O[i+1:], ok
```

Comment:
it should be checking if `ok := strings.HasPrefix(tempDB.O, "__TiDB_BR_Temporary_")`, right? And the `i+1` is the length of that temporary string.

Reply:
fixed this method